lib/manager/nuget/extract.spec.ts

```diff
@@ -215,5 +215,51 @@ describe('manager/nuget/extract', () => {
         expect(await extractPackageFile('{{', packageFile, config)).toBeNull();
       });
     });
+
+    describe('dotnet-tools.json', () => {
```

Comment:
I don't think we need to duplicate all those tests.

@rarkins I think the existing tests are enough. WDYT?

Reply:
@dignite PLease remove these new tests, no benefit but overall test slowdown.